src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -55,10 +55,11 @@ public sealed class NodeHandler : Handler, INodeHandler
         private const string nodeFolder = "node";
         private const string node10Folder = "node10";
         private const string node16Folder = "node16";
+        private const string node20Folder = "node20";
         private const string nodeLTS = node16Folder;
```

Comment:
Do we need to update nodeLTS?

Reply:
Node 20 isn't considered LTS until 24th October[^1]

[^1]: https://nodejs.dev/en/about/releases/